packages/calcite-components/src/components/tab/tab.e2e.ts

```diff
@@ -18,43 +17,7 @@ describe("calcite-tab", () => {
     defaults("calcite-tab", [
       { propertyName: "tab", defaultValue: undefined },
       { propertyName: "selected", defaultValue: false },
-      { propertyName: "scale", defaultValue: undefined },
+      { propertyName: "scale", defaultValue: "m" },
     ]);
   });
-
-  describe("when nested within calcite-tabs component", () => {
-    it("should render with medium scale", async () => {
-      const page = await newE2EPage({
-        html: `<calcite-tabs>${tabHtml}</calcite-tabs>`,
-      });
-      const element = await page.find("calcite-tab");
-      expect(element).toEqualAttribute("scale", "m");
-      expect(await (await element.getComputedStyle())["font-size"]).toEqual("14px");
```

Comment:
Can you confirm these are covered by screenshot tests? If not, let's make sure to add coverage.

Reply:
These are some stories we have that cover different scale scenarios. 
bordered
closable
withIcons
setWidth
justTabNav

These 2 screenshots are stories with knobs (including scale) actually that have `_TestOnly` suffix. Technically we can use them to check more cases with scale by removing the suffix when testing. Otherwise, we'd have to rewrite them to run screenshots for each scale, currently, the output is `m`.

simpleDarkModeRTL_TestOnly 
borderedDarkModeRTL_TestOnly

Reply:
If existing stories provide the same coverage that the E2E tests focused on, no further action is required. Otherwise, let's update existing tests to make sure we don't sacrifice coverage provided by removed E2E tests.

> Technically we can use them to check more cases with scale by removing the suffix when testing.

This would not be covered by automated testing though, right?

Reply:
Not specifically for dark mode/RTL/bordered combinations